edit/workload/index.vue

```diff
@@ -181,11 +182,19 @@ export default {
       podFsGroup:        podTemplateSpec.securityContext?.fsGroup,
       savePvcHookName:   'savePvcHook',
       tabWeightMap:      TAB_WEIGHT_MAP,
+      formRulesets:      [{
```

Comment:
You should only ever need the one formRulesets array, values in a sub-form should still have a value observable from the parent form and can thus be targeted by the `path` in the formRuleset object. In the rare case where the root object for the path requiring validation is not the parent form's `value` then it can be retargeted to the correct rootObject using the optional `rootObject` property on the formRuleset object.